src/main/java/ebig/han/nl/HelloController.java

```diff
@@ -8,7 +8,7 @@
 
     @RequestMapping("/")
     public String index() {
-        return "Greetings from my second continous deployment pipeline";
+        return "This message comes from a branch.";
```

Comment:
I do not accept this change. It should be "The origin of this message is a branch"